documentation/docs/references/web-workers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Web workers"
+path: /web-workers
+---
+
+# Web workers
+
+A web worker is a mechanism for running background tasks outside of the browser context. Heavy and resource consuming tasks can instead be moved to a background task so as to not block any UI threads in the main window. Communication is done through `postMessage` and `onmessage` handlers on the worker instance.
+
+Fusion.js supports loading web workers using the [`workerUrl` virtual module](/api/fusion-core#virtual-modules) exposed by `fusion-core`. The virtual module allows Fusion.js to bundle your service worker with Babel.
```

Comment:
> bundle your service worker

do you mean web worker? (also applies to service worker but that's a special case)